packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -2488,6 +2488,52 @@ void main() {
     expect(tester.getRect(find.byKey(icon0)), const Rect.fromLTRB(104.0, 562.0, 204.0, 582.0));
     expect(tester.getRect(find.byKey(icon1)), const Rect.fromLTRB(504.0, 562.0, 604.0, 582.0));
   });
+
+  testWidgets('BottomNavigationBar linear landscape layout label RenderFlex overflow',(WidgetTester tester) async {
+    tester.binding.window.physicalSizeTestValue = const Size(540, 340);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            return Scaffold(
+                bottomNavigationBar: BottomNavigationBar(
+              type: BottomNavigationBarType.fixed,
+              landscapeLayout: BottomNavigationBarLandscapeLayout.linear,
+              items: const <BottomNavigationBarItem>[
+                BottomNavigationBarItem(
+                  icon: Icon(Icons.home_rounded),
+                  label: 'Home Challenges',
+                  backgroundColor: Colors.grey,
+                  tooltip: '',
+                ),
+                BottomNavigationBarItem(
+                  icon: Icon(Icons.date_range_rounded),
+                  label: 'Daily Challenges',
+                  backgroundColor: Colors.grey,
+                  tooltip: '',
+                ),
+                BottomNavigationBarItem(
+                  icon: Icon(Icons.wind_power),
+                  label: 'Awards Challenges',
+                  backgroundColor: Colors.grey,
+                  tooltip: '',
+                ),
+                BottomNavigationBarItem(
+                  icon: Icon(Icons.bar_chart_rounded),
+                  label: 'Statistics Challenges',
+                  backgroundColor: Colors.grey,
+                  tooltip: '',
+                ),
+              ],
+            ));
+          },
+        ),
+      ),
+    );
+
```

Comment:
Shouldn't this test be verifying that the expected ellipsis suffixes have appeared?

Reply:
I am not sure because the ellipsis was already applied, should we verify that the `overflow` property is `TextOverflow.ellipsis`? Can you please guide me with this?

Reply:
I think you need to verify that the "..." suffix actually appears as expected.

Reply:
I don't know how to verify that. I have looked into it and couldn't find any way except golden test. The current test verifies that there will be no render overflow. I am kind of stuck here.